app/dragontts/app/cache/service.py

```
@@ -0,0 +1,1681 @@
+                )
+            aligned = bool(cwords) and len(cwords) == len(norm_words)
+        except Exception as e:
+            logger.warning(f"timestamped synth failed for warm ({e}); full-phrase store")
+            result = await self._synthesize(req, provider, model)
+            audio = result.audio
+
+        rate, bps = 16000, 2
+        clip_end = len(audio) / (rate * bps)
+
+        def byte_off(t: float) -> int:
+            return int(round(t * rate * bps))
```

Ruff (0.15.21)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)